docs/project/contribution_tools.md

```diff
@@ -239,6 +240,44 @@ considering if they fit your workflow.
         ```
         -   **NOTE**: This assumes you have `python` 3 installed on your system.
 
+#### Using LLDB with VS Code
+
+The required setup for LLDB is:
+
+1.  Install a minimum of LLVM 19 instead of LLVM 16.
+    -   The `lldb-dap` tool was added as part of LLVM 19.
+2.  Symlink `lldb_launch.json` to `launch.json`:
+    `ln -s lldb_launch.json .vscode/launch.json`
```

Comment:
Could we make it a bit more explicit what this is doing? Thinking about something like this:

```suggestion
2.  Symlink `lldb_launch.json` to `launch.json` in the workspace directory (typically the checkout of the `carbon-lang` repository):
    `ln -s lldb_launch.json .vscode/launch.json`
```

Reply:
How about instead writing "In the `.vscode` subdirectory"?